client/blocks/reader-post-card/index.jsx

```diff
@@ -146,7 +146,7 @@ export default class RefreshPostCard extends React.Component {
 
 		return (
 			<Card className={ classes } onClick={ this.handleCardClick }>
-				<PostByline post={ post } site={ site } feed={ feed } showSiteName={ showSiteName } />
+				<PostByline post={ originalPost ? originalPost : post } site={ site } feed={ feed } showSiteName={ showSiteName } />
```

Comment:
Should pass the byline the original post in addition to the post here, so we can use data from both.